src/server/wesnothd/server.hpp

```diff
@@ -207,6 +207,7 @@ class server : public server_base
 	void stats_handler(const std::string &, const std::string &, std::string &, std::ostringstream *);
 	void metrics_handler(const std::string &, const std::string &, std::string &, std::ostringstream *);
 	void requests_handler(const std::string &, const std::string &, std::string &, std::ostringstream *);
+	void roll_handler(const std::string&, const std::string&, std::string&, std::ostringstream*);
```

Comment:
On the other lines there's a space between the base type and the reference or pointer modifier. (`std::string &` rather than `std::string&`)

Reply:
Sorry, the autoformatter.